### ISSUE TITLE
The future size of the repository due to documentation

### ISSUE THREAD

**Issue by wwalc** [ckeditor/ckeditor5#516]
Now this is a tricky question, but came to my head. With time, the number of images in documentation will start increasing. An average PNG image with good quality, even processed by PNG compressors, takes like 400KB. We are at the very beginning of editors life, so it may happen that the same image (with the same topic/scenario) will get replaced several times. Also we have a plan to start adding animated gifs and so on. It means that the CKEditor 5 repository may get heavy because of docs. 

Let's have a look at CKEditor 4. The guides folder with images takes 8.7M, however in CKEditor 4 docs we did not add animated gifs. In any case, currently the `.git` folder of CKEditor 4 documentation takes already 75M (because of images being added, deleted and so forth). 

So, to sum up, are we okay if this repo (I'm talking about `.git` folder) grows up to 100+MB with time, just because of images?

**Comment by Reinmar** [comment]
I wouldn't mind if the `.git` folder had 100MB or even 1GB. The thing which counts is the initial clone time, but you clone `ckeditor5` very rarely – usually once. And in the whole "bootstrap" process cloning is in fact just a small step – the `mgit bootstrap && lerna bootstrap` take a lot more time.

OTOH, if we are able to save this repository from growing that would be nice. But where could we place those images? You need them when generating docs anyway, so we'd need to make them available for that. Which means that most of us will clone that anyway. So, I'd just keep them here and hope that we're not going to exceed some reasonable size.

BTW, I checked that ckeditor-docs is 11MB, ckeditor-dev is 45MB, Angular is already 62MB and I stopped cloning React at 10% and 50MB downloaded.

**Comment by wwalc** [comment]
Ops, true. The `.git` folder of a fresh clone of `ckeditor-docs` takes 11MB. The 75MB is just a result of poor garbage collection in a directory which has been in use for many years. That (11MB being the real size of `ckeditor-docs`) makes this issue pointless I guess.
